### ISSUE TITLE
VMs in shutoff not displaed on web interface

### ISSUE THREAD

**Issue by rseuster** [hep-gc/cloudscheduler#153]
I thought we talked about it, is it only in dev ?

There's 125 VMs on cc-west, but the web interface only shows 122 VMs in running state and
2 VMs in retiring. One is in SHUTDOWN state:
```
[seuster@csv2 moni]$ cloudscheduler vm list -g atlas -cn cc-west-a -s prod -V status | grep -v ACTIVE
Server: prod, Active User: <sqlalchemy.ext.automap.csv2_user object at 0x7f3b15d8ec88>, Active Group: atlas, User's Groups: ['atlas', 'belle', 'coca', 'test', 'testing']

VMs:
+-------+-----------+-------------------------------------------+---------+
+ Group | Cloud     | Hostname                                  | Status  +
+-------+-----------+-------------------------------------------+---------+
| atlas | cc-west-a | atlas--cc-west-a--1685--196002626408192   | SHUTOFF |
+-------+-----------+-------------------------------------------+---------+
Rows: 125
```

This VM should be in "ERROR" on the web display.

**Comment by kcasteels** [comment]
I think this is a db view issue. Can you have a look at this Colin?

**Comment by crlb** [comment]
fixed 531e610add7b20c797dfa6d55faa720ec24a86cb
